### ISSUE TITLE
[DataCap Application] <Genesis> - <SDCloud>

### ISSUE THREAD

**Issue by 1ane-1** [filecoin-project/filecoin-plus-large-datasets#465]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: (Genesis) This is the same project from https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/260 
Nobody reply to me for the next setp so i apply for it again.
- Website / Social Media:chuangshiIPFS-slack
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PIB
- Weekly allocation of DataCap requested (usually between 1-100TiB):100T
- On-chain address for first allocation:f3tdxw5l45xgjgfrbhfkt25kenw7elodm7gnj7bvs75trvlxgvz6ealrkxkn3eqtfpf462rpp4v36kyiml47ea

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
SDCloud Based on NEAR public chain, combined with distributed storage technology IPFS and Filecoin, data is stored hot and cold.Develop smart contracts using Rust language on NEAR, where users can select miners for data trading;Users can pay for their own storage by paying NEAR without having to pay Fil for data storage;Users can pay a small single fee to store their hot data on the IPFS network; In terms of cold data storage, pay NEAR to store data on Filecoin network according to storage miner quote, file size, storage cycle, etc.Applications that allow users to store encrypted data on Filecoin can only be decrypted by the user using the NEAR private key, thus maintaining ownership of their data;You can start storing your files in minutes. Filecoin offers persistent, decentralized storage - guaranteeing the integrity of the underlying data through a robust economic model while also providing cryptographic proofs for verifiability. IPFS offers content addressing. Write smart contracts in Rust on NEAR, you only pay a small fee for NEAR as file storage. Support multiple file type preview function.The English name of our product is SDCloud, and the Chinese name is shangding cloud. The link URL of the Chinese version is https://www.sdcloud.top/. We currently have PC terminals and Android terminals. SDCloud comes from Genesis group with 80 staffs in China. All the developers have rich experience.They dedicated to providing low-cost, high-security, and strong privacy storage solution.
```

What is the primary source of funding for this project?
```
It is funded by ourself.
```

What other projects/ecosystem stakeholders is this project associated with?
```
The project is associated with NEAR public chain.
```

## Use-case details

Describe the data being stored onto Filecoin
```
They are mainly surveillance video,pictures,music, documents,disaster recovery data，driverless data and some data from slingshot dataset.The current data volume is about 3-5T per day, which is why we apply for 5PIB datacap. In order to ensure that the data is healthy and compliant and protect the copyrights of users, we have established a special compliance KYC audit mechanism to ensure that our platform is green and healthy, so the data we upload to the filecoin network is also legal and reasonable. At the same time, these data can also be used to continuously grow the filecoin community and enrich the types of data storage.
```

Where was the data in this dataset sourced from?
```
We build a platfrom named SDCloud that users can upload their data on it.So our dataset mainly comes from our users and some other enterprise and government.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
(http://122.114.37.226:6580/)
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Yes, we confirm that the dataset is public and can be retrieved by anyone.
```

What is the expected retrieval frequency for this data?
```
It won't be retrived very often because of amout of data.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
3 years at least, if needed we will keep for a long time.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
Asia
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Both online and offline transfer process are OK.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We pay attention to their work time and how much they cost.  We need comprehensive storage providers who can coorperate for a long time.
```

How will you be distributing deals across storage providers?
```
We will distribute different deals according to their distance. 
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
We're ready for making deals.
```

**Comment by 1ane-1** [comment]
> # Large Dataset Notary Application
> To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.
> 
> ## Core Information
> * Organization Name: Genesis
> * Website / Social Media:chuangshiIPFS-slack
> * Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PIB
> * Weekly allocation of DataCap requested (usually between 1-100TiB):100T
> * On-chain address for first allocation:f3tdxw5l45xgjgfrbhfkt25kenw7elodm7gnj7bvs75trvlxgvz6ealrkxkn3eqtfpf462rpp4v36kyiml47ea
> 
> _Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._
> 
> ## Project details
> Share a brief history of your project and organization.
> 
> ```
> SDCloud Based on NEAR public chain, combined with distributed storage technology IPFS and Filecoin, data is stored hot and cold.Develop smart contracts using Rust language on NEAR, where users can select miners for data trading;Users can pay for their own storage by paying NEAR without having to pay Fil for data storage;Users can pay a small single fee to store their hot data on the IPFS network; In terms of cold data storage, pay NEAR to store data on Filecoin network according to storage miner quote, file size, storage cycle, etc.Applications that allow users to store encrypted data on Filecoin can only be decrypted by the user using the NEAR private key, thus maintaining ownership of their data;You can start storing your files in minutes. Filecoin offers persistent, decentralized storage - guaranteeing the integrity of the underlying data through a robust economic model while also providing cryptographic proofs for verifiability. IPFS offers content addressing. Write smart contracts in Rust on NEAR, you only pay a small fee for NEAR as file storage. Support multiple file type preview function.The English name of our product is SDCloud, and the Chinese name is shangding cloud. The link URL of the Chinese version is https://www.sdcloud.top/. We currently have PC terminals and Android terminals. SDCloud comes from Genesis group with 80 staffs in China. All the developers have rich experience.They dedicated to providing low-cost, high-security, and strong privacy storage solution.
> ```
> 
> What is the primary source of funding for this project?
> 
> ```
> It is funded by ourself.
> ```
> 
> What other projects/ecosystem stakeholders is this project associated with?
> 
> ```
> The project is associated with NEAR public chain.
> ```
> 
> ## Use-case details
> Describe the data being stored onto Filecoin
> 
> ```
> They are mainly surveillance video,pictures,music, documents,disaster recovery data，driverless data and some data from slingshot dataset.The current data volume is about 3-5T per day, which is why we apply for 5PIB datacap. In order to ensure that the data is healthy and compliant and protect the copyrights of users, we have established a special compliance KYC audit mechanism to ensure that our platform is green and healthy, so the data we upload to the filecoin network is also legal and reasonable. At the same time, these data can also be used to continuously grow the filecoin community and enrich the types of data storage.
> ```
> 
> Where was the data in this dataset sourced from?
> 
> ```
> We build a platfrom named SDCloud that users can upload their data on it.So our dataset mainly comes from our users and some other enterprise and government.
> ```
> 
> Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this.
> 
> ```
> (http://122.114.37.226:6580/)
> ```
> 
> Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
> 
> ```
> Yes, we confirm that the dataset is public and can be retrieved by anyone.
> ```
> 
> What is the expected retrieval frequency for this data?
> 
> ```
> It won't be retrived very often because of amout of data.
> ```
> 
> For how long do you plan to keep this dataset stored on Filecoin?
> 
> ```
> 3 years at least, if needed we will keep for a long time.
> ```
> 
> ## DataCap allocation plan
> In which geographies (countries, regions) do you plan on making storage deals?
> 
> ```
> Asia
> ```
> 
> How will you be distributing your data to storage providers? Is there an offline data transfer process?
> 
> ```
> Both online and offline transfer process are OK.
> ```
> 
> How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
> 
> ```
> We pay attention to their work time and how much they cost.  We need comprehensive storage providers who can coorperate for a long time.
> ```
> 
> How will you be distributing deals across storage providers?
> 
> ```
> We will distribute different deals according to their distance. 
> ```
> 
> Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
> 
> ```
> We're ready for making deals.
> ```

**Comment by Kakkouii** [comment]
Take a look at this.
https://github.com/filecoin-project/notary-governance/issues/411#issuecomment-1120294025

**Comment by 1ane-1** [comment]
> 

Thanks for your question, pls let me explain it to you. Here about what Neo's comments that we have alreay try our best to correct it and made a clarifytion to Kevin and Deep and Galen. You can check this link: https://filplus.d.interplanetary.one/clients/f0510343/breakdown

**Comment by 1ane-1** [comment]
> Take a look at this. [filecoin-project/notary-governance#411 (comment)](https://github.com/filecoin-project/notary-governance/issues/411#issuecomment-1120294025)

Thanks for your question, pls let me explain it to you. Here about what Neo's comments that we have alreay try our best to correct it and made a clarifytion to Kevin and Deep and Galen. You can check this link: https://filplus.d.interplanetary.one/clients/f0510343/breakdown

**Comment by Kakkouii** [comment]
Yeah there're 4 SPs now but it's very clear that f01019009 has less than 1% of total data let alone the rest 3 nodes are exceeding the data limitation of Fil+. @neogeweb3
<img width="1329" alt="image" src="https://user-images.githubusercontent.com/105268629/177391542-273df528-0a13-4c4d-826f-e4f3a0b49de4.png">

**Comment by neogeweb3** [comment]
The [flow chart](https://ipfs.io/ipfs/QmdseAWQFVm8ntLZ5mkHyYRs6bB3g2LACT44wjB5zPv4rq) is clear: I blacklist the client and flag the community, when severe violations occur. After that, it depends on the RKHs and community on how to handle it. In this case, RKHs decided to pause her notary application but then continued to process and granted round 3 notary identity. Even though the extra measures to ensure there will be no self-dealing from this notary in the future are unclear, which brought me some concerns, I do see some improvement in DataCap distribution for diversity. 

With that being said, I have finished my part as a round 1 notary, tagging me will unlikely trigger any further investigations.

**Comment by 1ane-1** [comment]
> Kakkouii

Thanks for the problem pointed out by NEO in the early stage, we have corrected the error in time after learning that the problem occurred, and disclosed the latest progress of the incident under the corresponding problem.
1. The content you screenshot is because the existing Datacap has been used up. We have tried our best to avoid the self-trading rules, so the content shown in the screenshot appears, but we will ensure that it is lower than 25% after we have enough Datacap. I also ask you and the community to monitor this. The filecoin community is a global open source community. Helping each other is part of the community culture, and it is also one of the charms of filecoin's success;
2. Regarding the problem of self-trading rules, we have communicated with Kevin,Galen,Deep and other responsible persons separately, admitted mistakes and corrected the problems in time. At the same time, we are also actively contributing to the ecological implementation of the project; we have successfully participated in community organizations. 'filecoin hackathon', 'SPX bounty program', 'Slingshot 2', 'FVM early builder program' and other projects, contribute your little effort. At the same time, we independently developed the file storage software 'sdcloud (Shang Ding Cloud)' project to speed up the implementation of the filecoin storage project.
3. Here, we would like to thank Neo, Kakkouii and community members for supervising us. We are willing to move forward with the filecoin project and assume the obligations and responsibilities of notaries. I sincerely hope that filecoin can bring more changes to our lives!
